ppa/archive/admin.py

```diff
@@ -6,9 +6,13 @@
 
 
 class DigitizedWorkAdmin(admin.ModelAdmin):
-    list_display = ('source_id', 'title', 'author', 'enumcron', 'pub_place',
-        'publisher', 'pub_date', 'page_count', 'added', 'updated')
-    readonly_fields = ('source_id', 'source_url', 'page_count',
+    list_display = ('title', 'hathitrust_link', 'author', 'list_collections',
```

Comment:
Minor quibble: I've been trying to name things generically (even though we're only working with hathi materials for now) so it will be easier to add other content when we get to that point.  I don't feel strongly about holding this up for a rename though.

Reply:
@rlskoeser I can make the change quickly. Do you have thoughts on the name, i.e. `source_link`?